packages/core/src/diff/input.ts

```diff
@@ -30,6 +35,15 @@ export function changesInInputObject(
       changesInInputField(oldInput, field.oldVersion, field.newVersion, addChange);
     },
   });
+
+  compareLists(oldInput.astNode?.directives || [], newInput.astNode?.directives || [], {
+    onAdded(diractive) {
+      addChange(inputObjectAdded(newInput, diractive));
```

Comment:
should this be something else? or is it just me being confused by the name `inputObjectAdded` and `inputObjectRemoved`?

Reply:
Also, `directive` instead of `diractive`